src/test/resources/feature/checkout2-1.feature

```diff
@@ -0,0 +1,25 @@
+Feature: complete order with default values
+  Background:User is logged in and has items in the cart
```

Comment:
Please divide this step in
 **user is logged in** 
and 
**user has items in the cart** 
If you will let it like this and your step will fail, you won't know why.

Reply:
Done